### ISSUE TITLE
Build fails due to lack of go-ethereum/ethapi

### ISSUE THREAD

**Issue by jakubgs** [status-im/status-cli-client#1]
I'm trying to build this but all I'm getting is this:
```
 $ go build -v
...
build github.com/status-im/status-cli-client: cannot load github.com/ethereum/go-ethereum/ethapi: cannot find module providing package github.com/ethereum/go-ethereum/ethapi
```

**Comment by jakubgs** [comment]
I checked in `status-protocol-go` and it's not using this non-existent `ethapi` folder:
https://github.com/status-im/status-protocol-go/search?q=ethapi&unscoped_q=ethapi
But if I look at `status-go` it actually does:
https://github.com/status-im/status-go/search?q=ethapi&unscoped_q=ethapi
Here:
https://github.com/status-im/status-go/blob/e118e891070378ced4eab5f506fc241b5d24b3f3/vendor/modules.txt#L56
And here:
https://github.com/status-im/status-go/blob/b003400b4b658078bb7bde4d6bcfcd3667a48f87/services/personal/service.go#L5
Which does exist in `vendor` folder:
https://github.com/status-im/status-go/tree/e118e891070378ced4eab5f506fc241b5d24b3f3/vendor/github.com/ethereum/go-ethereum/ethapi
Not sure what that's about.

**Comment by jakubgs** [comment]
Another weird thing that I don't get is that [`status-console-client`](https://github.com/status-im/status-console-client) is using a `status-protocol-go` commit that doesn't exist:
https://github.com/status-im/status-console-client/blob/master/go.mod#L41
Just look:
https://github.com/status-im/status-console-client/commit/57675d19152c73af930b1e8f93d298322cbfe95c
But it does exist in the `vendor` folder:
https://github.com/status-im/status-console-client/blob/master/vendor/modules.txt#L363
https://github.com/status-im/status-console-client/tree/master/vendor/github.com/status-im/status-protocol-go
This is seriously annoying.

**Comment by jakubgs** [comment]
I really fucking hate these retarded `vendor` folders. They make it too easy for repo owners to fuck up their repos by force-pushing and other ways without anyone noticing until much, much later.

**Comment by jakubgs** [comment]
What I don't get, is that the only parts of `status-go` I'm using are these:
https://github.com/status-im/status-cli-client/blob/de4d5f678461b022209436f3e3dd28f4c9dd3d5e/main.go#L13-L14
And `ethapi` is not being used in those folders:
```
~/work/status-go develop ⇣
 lilim > cd node 

~/work/status-go/node develop ⇣
 lilim > ag ethapi

~/work/status-go/node develop ⇣
 lilim > cd ../params 

~/work/status-go/params develop ⇣
 lilim > ag ethapi   

```
And yet it fails on that folder.

**Comment by jakubgs** [comment]
I tried versions of `status-protocol-go` from `v0.1.0` to `v0.1.4`, all fail with the same error.
As far as I can tell this is an issue with `status-go` rather than `status-protocol-go`.

@adambabik any ideas?

**Comment by jakubgs** [comment]
Adam has fixed this issue in #2 PR.

Though when i try his branch(after fixing the commit SHA1) I get something new:
```
failed to apply mvds migrations: failed to migrate: no such table: mvds_schema_migrations in line 0: DELETE FROM mvds_schema_migrations
```
I've opened https://github.com/status-im/status-protocol-go/issues/65 in regards to that.

**Comment by jakubgs** [comment]
The migration bug was fixed here - https://github.com/status-im/migrate/commit/a9d340ec8fb76af4afda06acf01740d45d2661ed - by Adam.
